src/traces/table/plot.js

```diff
@@ -0,0 +1,82 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var table = require('./render');
+
+module.exports = function plot(gd, cdTable) {
```

Comment:
@monfera I'm not a fan of your `plot.js` + `render.js` pattern. I know it makes `render.js` somewhat plotly-agnostic, but it also makes very painful to patch. You couldn't believe how much I was :rage:-ing while working on https://github.com/plotly/plotly.js/pull/1896

So, could you merge `render.js` into `plot.js` to make `table/` look a little more like the other traces?